test/blackbox/common/DDSBlackboxTestsDiscovery.cpp

```diff
@@ -330,6 +330,124 @@ TEST(DDSDiscovery, AddDiscoveryServerToListTCP)
     server_2.wait_discovery(std::chrono::seconds::zero(), 2, true); // Knows client1 and server1
 }
 
+TEST(DDSDiscovery, ServersConnectionTCP)
+{
+    using namespace eprosima;
+    using namespace eprosima::fastdds::dds;
+    using namespace eprosima::fastrtps::rtps;
+
+    // TCP default DS port
+    std::string W_UNICAST_PORT_RANDOM_NUMBER_STR = "42100";
```

Comment:
I think this can directly be an `uint16_t`:
```suggestion
    constexpr uint16_t W_UNICAST_PORT_RANDOM_NUMBER_STR = 42100;
```